pom.xml

```diff
@@ -292,6 +292,17 @@ The MIT License (MIT)
                   <placement>X</placement>
                 </tag>
               </tags>
+              <reportOutputDirectory>${project.basedir}/target</reportOutputDirectory>
+              <destDir>javadoc</destDir>
+              <show>private</show>
+              <detectLinks>false</detectLinks>
+              <detectOfflineLinks>true</detectOfflineLinks>
+              <linksource>true</linksource>
+              <detectJavaApiLink>true</detectJavaApiLink>
```

Comment:
@scrobot why do we need all thos new options? Please remove everything that is not needed